### PR TITLE
Name the location of backups at the top of the docs page

### LINKED ISSUE #33564: The backup page doesn't show how to configure backups? Where are they stored to?
### Feedback

Should provide a link at least, I'm using a docker container to run HA.

### URL

https://www.home-assistant.io/integrations/backup/

### Version

2024.7.0

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by j4n-e4t**
## Proposed change

Explain, wehre HA created the backup .tar at the top of the page, not just in the "Restore" section to make it more clear where the backups are located.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: /
- Link to parent pull request in the Brands repository: /
- This PR fixes or closes issue: fixes #33564

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Clarified instructions on backup creation and usage for different Home Assistant installations.
  - Added details on backup storage locations for Home Assistant Core and Container installations.
  - Included an important note directing users of Home Assistant Operating System or Supervised installations to built-in backup documentation.
  - Streamlined recovery steps by removing redundant information for Home Assistant Container installations.

### CODE REVIEW

File: source/_integrations/backup.markdown [hunk 1]
```diff
@@ -21,9 +21,12 @@ related:
     title: Home Assistant Operating System- Restoring a backup
 ---
 
-The **Backup** {% term integration %} is used for {% term "Home Assistant Core" %} and {% term "Home Assistant Container" %} installations to create and download backups. This backup file can be used if you migrate to {% term "Home Assistant Operating System" %}.
+The **Backup** {% term integration %} is used for {% term "Home Assistant Core" %} and {% term "Home Assistant Container" %} installations to create and download backups, which are stored in a subdirectory `backups`, located in your [configuration directory](/docs/configuration/#editing-configurationyaml).
```
Comment (frenck) [inline source/_integrations/backup.markdown:24]: I don't think this is fitting for the lead paragraph. Let's add some additional context in a separate paragraph.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/backup.markdown (hunk 1 above)
Comment (j4n-e4t) [reply source/_integrations/backup.markdown:24]: Yes, you're right. That indeed makes it even better 👍
Screenshot 2024-07-11 at 23 11 32

Is the PR now ready to merge or are still changes needed?

File: source/_integrations/backup.markdown [hunk 2]
```diff
@@ -21,9 +21,12 @@ related:
...
+This backup file can be used if you migrate to {% term "Home Assistant Operating System" %}.
 
-Note: If you use {% term "Home Assistant Operating System" %} or {% term "Home Assistant Supervised" %} installation, this page is not for you. Instead, refer to the documentation on using the [built-in back up](/common-tasks/os/#backups).
+{% important %}
+If you use {% term "Home Assistant Operating System" %} or {% term "Home Assistant Supervised" %} installation, this page is not for you. Instead, refer to the documentation on using the [built-in back up](/common-tasks/os/#backups).
+{% endimportant %}
```
[the remark this answers is not in the document]
Comment (j4n-e4t) [reply source/_integrations/backup.markdown:29]: This is not relevant for my PR, right?

**Comment by frenck** [comment]
Converted back to draft, as the review comment above wasn't addressed or picked up.

File: source/_integrations/backup.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/backup.markdown:24]: I don't think it makes it better, as per above.

The suggestion was to move it into a different paragraph and out of the lead text.

**Pull request closed without merging**